addons/website_mass_mailing/static/src/js/website_mass_mailing.editor.js

```diff
@@ -148,4 +149,18 @@ WysiwygMultizone.include({
         return Promise.all(defs);
     },
 });
+
+WysiwygTranslate.include({
+    /**
+     * @override
+     */
+    start: function () {
+        this.$target.on('click.newsletter_popup_option', '.o_edit_popup', function (ev) {
+            // So that the snippet is not enabled again by the editor
+            alert(_t('Website popups can only be translated trough the Email Marketing app. You can access it by going to Email Marketing > Mailing Lists > List > ... | Edit > Website Popups.'));
```

Comment:
```
Email Marketing > Mailing Lists > List > ... | Edit > Website Popups.
```

is a little too specific, these menu / helps are often renamed (and this probably would not be adapted here). I would just say something like:

> Website popups can only be translated trough mailing configuration in the Email Marketing app.